core/includes/install.core.inc

```diff
@@ -1832,6 +1839,7 @@ function _install_configure_form($form, &$form_state, &$install_state) {
   $form['admin_account']['account']['mail'] = array(
     '#type' => 'email',
     '#title' => st('E-mail address'),
+    '#description' => st("Notifications about pending security upadtes and site problems will be sent to this email address. It can be an address hosted on an external domain (such as a gmail.com or hotmail.com email)."),
```

Comment:
```
It can be an address hosted on an external domain (such as a gmail.com or hotmail.com email)
```

This adds no value, if there are no restrictions we don't need to tell people there are no restrictions. All it does is add words, which makes people less likely to read the text.